salt/modules/smtp.py

```diff
@@ -106,22 +108,29 @@ def send_msg(
         sender = creds.get("smtp.sender")
         username = creds.get("smtp.username")
         password = creds.get("smtp.password")
+        port = creds.get("smtp.port", 0)
 
     if attachments:
         msg = email.mime.multipart.MIMEMultipart()
-        msg.attach(email.mime.text.MIMEText(message))
+        if is_html:
+            msg.attach(email.mime.text.MIMEText(message, "html"))
+        else:
+            msg.attach(email.mime.text.MIMEText(message))
     else:
-        msg = email.mime.text.MIMEText(message)
+        if is_html:
+            msg = email.mime.text.MIMEText(message, "html")
+        else:
+            msg = email.mime.text.MIMEText(message)
```

Comment:
**question** would this read nicer as:

```
msg.attach(email.mime.text(MIMEText(message, "html" if is_html else "plain"))
```



And,

```
msg = email.mime.text.MIMEText(message, "html" if is_html else "plain")

```

?

Reply:
Alternatively, just expose the `email_subtype` (or some other name) and default it to `"plain"`